consoles/ssh_screen.pm

```diff
@@ -61,7 +61,7 @@ sub type_string ($self, $nargs) {
     while ($written < length($text)) {
         my $elapsed = consoles::serial_screen::elapsed($stime);
 
-        croak("type_screen(): Timed out after $elapsed seconds.")
+        croak("(caller(0))[3]: Timed out after $elapsed seconds.")
```

Comment:
I doubt you really want to output `(caller(0))[3]` literally here - so the expression should not be within a string literal.

Reply:
*this change is ready for review*